src/BoilerPlate/Game.cpp

```diff
@@ -464,3 +480,26 @@ void Game::DrawFrameRateMeter()
 	}
 	glEnd();
 }
+
+void Game::RenderLives()
+{
+	
+	lifePosition = 0;
+
+	for (int i = 0; i < playerLife; i++)
+	{
+		glLoadIdentity();
+		glTranslatef((-(width / 2.0f) + 50.0f) + lifePosition, ((height / 2.0f) - 50.0f), 0.0f);
```

Comment:
This offset translation is cool, but move the values to constant. Avoid magic numbers!